### ISSUE TITLE
conversions are corrupted from png to webp :/

### ISSUE THREAD

**Issue by jonrrivera** [webponize/webponize#34]

**Comment by bobdowns** [comment]
Same here. App uninstalled after only one attempt at use. (Interface is not user-friendly as well.)

[MacBook Pro (15-inch, 2018), macOS Monterey Version 12.3.1]

**Comment by renjujv** [comment]
Yes. The image is getting corrupted after conversion from PNG to WebP. Let me know if there's any way I can share logs.

Source Image:
<img width="1220" alt="jenkins_login_page copy" src="https://user-images.githubusercontent.com/3957722/164975296-fc4ebfce-3177-4dd4-a119-2aaf769ebf9e.png">

After conversion:
<img width="1215" alt="Screenshot 2022-04-24 at 5 28 43 PM" src="https://user-images.githubusercontent.com/3957722/164975330-19a71c58-0e91-463c-81f6-8749d5876bf7.png">
Note: Attaching screenshot for WebP preview since WebP cannot be attached here.

**Comment by 1000ch** [comment]
@renjujv I could reproduce it with your attached image, but It worked well after I fixed the corruption of the original image using [ImageOptim](https://imageoptim.com/mac). I'm not sure about what happened to that processing..

**Comment by renjupauljosev** [comment]
Okay. I see ImageOptim is opensource. Do you use that internally or think we can?
